.gitignore

```diff
@@ -46,3 +46,6 @@ gha-creds-*.json
 
 ### jEnv
 .java-version
+
+### data directory for postgres in local development
+/var/
```

Comment:
When spinning up a local Postgres according to the main README.md, it creates this directory. Let's git-ignore it.